osf_tests/settings.py

```diff
@@ -22,11 +22,21 @@
 )
 
 REST_FRAMEWORK['ALLOWED_VERSIONS'] = (
-        '2.0',
-        '2.0.1',
-        '2.1',
-        '2.2',
-        '2.3',
-        '3.0',
-        '3.0.1',
-    )
+    '2.0',
+    '2.0.1',
+    '2.1',
+    '2.2',
+    '2.3',
+    '3.0',
+    '3.0.1',
+)
+REST_FRAMEWORK['DEFAULT_THROTTLE_CLASSES'] = ()
```

Comment:
Are these lines necessary, since we're already setting the high throttle rates in `local-travis.py`?